Assets/AnkrDemo/Scripts/HeaderWalletButton.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using AnkrSDK.Data;
+using AnkrSDK.Examples.UseCases.WebGlLogin;
+using AnkrSDK.WebGL.DTO;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Demo.Scripts
+{
+	public class HeaderWalletButton : MonoBehaviour
+	{
+		public WalletItem WalletItem;
```

Comment:
No punlic fields are allowed. If it should serialzied by unity use [SerializeField] if it has to be publicly accessible by other scripts use public property.

Reply:
I need this field for get and set operations from outside the script. Do I realy need to create some specialgetters and setters?

Reply:
Yes, you do. you need to encapsulate this field using public getter/setter

Reply:
https://blog.codinghorror.com/properties-vs-public-variables/